src/signer/canister/tests/it/utils/pic_canister.rs

```diff
@@ -0,0 +1,279 @@
+//! Common methods for interacting with a canister using `PocketIc`.
```

Comment:
Note: This file will eventually replace `pocketic.rs`.  However quite  a few tests have to be updated before we can get rid of pocketic.rs.